app/admin-exec.php

```diff
@@ -389,8 +382,20 @@
 } // END SECURITY
 
 // EMAIL
-if ($_SERVER['REQUEST_METHOD'] == 'POST' && isset($_POST['smtp_address'])) {
+if ($_SERVER['REQUEST_METHOD'] == 'POST' && isset($_POST['mail_method'])) {
+
+    $mail_method = $_POST['mail_method'];
```

Comment:
Thanks a lot! This would have been a nice code injection path ;-) Fixed in https://github.com/Athemis/elabftw/commit/9adf93996131ea4612e88c0f5ea22f5e59f07d66